t/unit/test_transport.py

```diff
@@ -639,112 +640,144 @@ def test_wrap_context(self):
 
     def test_wrap_socket_sni(self):
         # testing default values of _wrap_socket_sni()
-        sock = Mock()
         with patch('ssl.SSLContext') as mock_ssl_context_class:
-            wrap_socket_method_mock = mock_ssl_context_class().wrap_socket
-            wrap_socket_method_mock.return_value = sentinel.WRAPPED_SOCKET
+            sock = Mock()
+            context = mock_ssl_context_class()
+            context.wrap_socket.return_value = sentinel.WRAPPED_SOCKET
             ret = self.t._wrap_socket_sni(sock)
 
-        mock_ssl_context_class.load_cert_chain.assert_not_called()
-        mock_ssl_context_class.load_verify_locations.assert_not_called()
-        mock_ssl_context_class.set_ciphers.assert_not_called()
-        mock_ssl_context_class.verify_mode.assert_not_called()
-        wrap_socket_method_mock.assert_called_with(
-            sock=sock,
-            server_side=False,
-            do_handshake_on_connect=False,
-            suppress_ragged_eofs=True,
-            server_hostname=None
-        )
-        assert ret == sentinel.WRAPPED_SOCKET
+            context.load_cert_chain.assert_not_called()
+            context.load_verify_locations.assert_not_called()
+            context.set_ciphers.assert_not_called()
+            context.verify_mode.assert_not_called()
+
+            context.load_default_certs.assert_called_with(
```

Comment:
@matusvalo I have tests to cover the new code path already.

Reply:
yes I saw. But I was curious why integration test creating real tls connection to broker suit does not fail on master.

Reply:
They should only fail when you try to load system certs. I added new tests for this case.